master/getting-started/rkt/tutorials/basic.md

```diff
@@ -87,69 +67,51 @@ cat >/etc/rkt/net.d/10-calico-frontend.conf <<EOF
 EOF
 ```
 
-## 4. Create containers
+## 3. Create test container
 
-With the networks created, let's start some containers. We'll create a "frontend" 
-container on `calico-01` and a "backend" container on `calico-02`.
-Both containers will just be a `busybox` image running a simple HTTP daemon `httpd`
+With the networks created, let's start some containers. We'll create a  
+container on `calico-01` in `network1`, and then create containers on `calico-02` 
+in each network to check connectivity to the first container.  For this tutorial, 
+the container we create on `calico-01` will be a `busybox` image running a simple HTTP daemon `httpd`
 serving up the containers local filesystem over HTTP.
 
 ### On calico-01
 
-Create the "frontend" container.  Note that we include a suffix `:IP=192.168.0.0`, this
-is used to pass in the IP environment through to the frontend network plugin which
-Calico IPAM uses to assign a specific IP address.  This may be omitted, in which case
-Calico IPAM will automatically select an IP address to use from it's configured
-IP Pools - however, to simplify this worked we use fixed IP addresses.
+Create the container in `network1`.  Note that we include a suffix `:IP=192.168.0.0`, this
```

Comment:
Interesting, we could probably add label support by passing them through using these suffixes. Food for thought for later.